includes/wp-admin/class-welcome-fields.php

```diff
@@ -90,8 +90,8 @@ public static function render_welcome_header_section() {
 							stroke-dashoffset="<?php echo \esc_attr( 339.292 - ( 339.292 * $progress_percentage / 100 ) ); ?>" />
 				</svg>
 			</div>
-			<h2 class="activitypub-welcome-title"><?php \esc_html_e( 'Let&#8217;s keep going!', 'activitypub' ); ?></h2>
-			<p class="activitypub-welcome-subtitle"><?php \esc_html_e( 'It&#8217;s time to finish setting up your plugin.', 'activitypub' ); ?></p>
+			<h2 class="activitypub-welcome-title"><?php \esc_html_e( 'Welcome to the Fediverse!', 'activitypub' ); ?></h2>
+			<p class="activitypub-welcome-subtitle"><?php \esc_html_e( 'Let&#8217;s get your WordPress site connected to the Fediverse in just a few simple steps.', 'activitypub' ); ?></p>
```

Comment:
```suggestion
			<p class="activitypub-welcome-subtitle"><?php \esc_html_e( 'Complete your setup in just a few simple steps.', 'activitypub' ); ?></p>
```
How about tightening it up a little bit? "Fediverse" is already used in the title and "WordPress site" feels redundant.

Reply:
I liked the "connected"... but I am fine with removing the duplicate "fediverse"... either way, it still feels a bit short... I miss kind of a welcome/introduction thingy...

Something like we had before with:

> Enter the fediverse with <strong>ActivityPub</strong>, broadcasting your blog to a wider audience. Attract followers, deliver updates, and receive comments from a diverse user base on <strong>Mastodon</strong>, <strong>Friendica</strong>, <strong>Pleroma</strong>, <strong>Pixelfed</strong>, and all <strong>ActivityPub</strong>-compliant platforms.

Reply:
If you like connected, how about "Get connected in just a few steps." or "Follow these steps to get connected."?

They're also short I realize, but I doubt anyone actually reads anything longer. And it helps with keeping whitespace visually.

Reply:
ok, fair enough.